app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -94,6 +98,24 @@ public void moveItem(int fromPosition, int toPosition) {
         notifyItemMoved(fromPosition, toPosition);
     }
 
+    public void refresh() {
```

Comment:
Thanks, then simply we can have it in Bookmarks fragment only....and after refreshing the list, we will call `adapter.notifyDataSetChanged()`